### DISCUSSION TITLE
How to make sure the component always be triggered?

### DISCUSSION THREAD

**Discussion by supermpc** [primefaces/community#570] [PrimeVue]
Hi, 

 In PrimeVUE framework version: 3.41.1
----------------------------------
js code:
```
const aa = ref()
function test(event) {
console.log(1, event)
}
```
----------------------------------
The html code is as follows:
`<InputNumber v-model="aa" @input="test" />`
 
When input in the InputNumber component, sometimes the test function can be trigger , but sometimes not. Why? Is there something I didn't do right?

**Comment by FlipWarthog** [comment]
Are you able to put together a reproducer of what you're seeing? This sample seems to be working fine: https://stackblitz.com/edit/8qmowa?file=src%2FApp.vue
